### PR TITLE
Homework week2

### PR DESCRIPTION
**Pull request by AleOrtizA**
This is actually homework for week2

### CODE REVIEW

File: Week2/homework/map_filter.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 'use strict';
-
 const numbers = [1, 2, 3, 4];
 
-// Add your code here
+const newNumbers = numbers.filter(x => x % 2 !== 0).map(x => x * 2)
```
Comment (silksil) [inline Week2/homework/map_filter.js]: well done!

File: Week2/homework/maartjes_work.js [hunk 2]
```diff
@@ -43,5 +43,18 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+console.log(tasks);
```
Comment (silksil) [inline Week2/homework/maartjes_work.js]: once you have finished the exercise, try to remove unnecessary console.logs. it's make your code more clean

File: Week2/homework/maartjes_work.js [hunk 3]
```diff
@@ -43,5 +43,18 @@ const tuesday = [
...
+const fromMinToHours = tasks.map(tasks => tasks.duration / 60);
+console.log(fromMinToHours);
+
+const underTwoHours = tasks.filter(tasks => tasks.duration > 120);
+console.log(underTwoHours);
+
+const maartjeSalary = tasks.map((tasks) => tasks.duration * 20);
+console.log(maartjeSalary);
+
+const amount = tasks.reduce((maartjeSalary, tasks) => maartjeSalary + tasks.duration, 0);
```
Comment (silksil) [inline Week2/homework/maartjes_work.js]: try a bit more descriptive in naming your variables/functions. This is called amount, but you also have a functioned called maartjeSalary. I think 'amount'  is basically also calculating the salary of maartje, so why do you name this 'amount' and the other 'maartjeSalary'.

Instead, you could think of naming it something like:  salaryPerTask and the other totalSalary

File: Week2/homework/maartjes_work.js (hunk 2 above)
Comment (AleOrtizA) [reply Week2/homework/maartjes_work.js]: I will remember that, thank you.

**Pull request closed without merging**